docs/announcements/github-release-v0.1.86.md

```
@@ -0,0 +1,32 @@
+# 🚀 Release Highlights — v0.1.86 (2026-05-13)
+
+### 🧠 `bootstrap_subagent` Discriminator Is Now Functional
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)